src/main/frontend/components/block.cljs

```diff
@@ -728,9 +724,15 @@
                       :target "_blank"}
                   (map-inline config label))
 
-          (and (util/electron?) (config/local-asset? s))
+          (and (util/electron?)
+               (config/local-asset? s)
+               (string/starts-with? (string/triml full_text) "!"))
```

Comment:
Is it working for `org-mode`? Does org-mode has a "display" mode for images?
I remember it's something like `[[1.jpg]]`

Reply:
updated, see also updated comments above